### ISSUE TITLE
Next beta/feature release 

### ISSUE THREAD

**Issue by fm-sys** [cgeo/cgeo#13307]
Our [changelog](https://github.com/cgeo/cgeo/blob/master/main/res/raw/changelog_base.md) at master has grown quite a bit, and many small features and bug fixes are waiting to be released... IMHO it would be time for a small beta/release round, before we start adding new larger changes.

Is there anything which needs to be done before we can start a beta?

**Comment by moving-bits** [comment]
Thanks for opening this issue - wanted to have done it myself a couple of days ago, but am currently on very tight time budget...

I don't know of any urgent issues with our currently released version, and also nighly versions seems to run smoothly, so time for a beta seems to be right. 

Regarding the beta's contents I'm not aware of any urgent missing things, so we could start right now... But I would really like to see the "linear" info included, if possible. It seems to me like a rather small and uncritical change (at least detection, storing to database and displaying it in a textual way), but giving valuable info for advanced users. Displaying specific icons could be added in a later version, if needed.

Thus my proposal would be to wait for a linear PR, then branch, and use beta phase to check for any open issues.

**Comment by Lineflyer** [comment]
Same for me with the time budget :(
I can't even say if the nightlies are running smooth, but I fully trust the team. So no objection to get help of our beta community by bringing it to the beta state.

This would also allow starting with `master` to check if we are able to overcome the language issue by ourselves.

**Comment by fm-sys** [comment]
I'm using the nightlys on my main device. Although I do not test every new feature in detail, they seem to run smoothly overall...

**Comment by moving-bits** [comment]
If no one stops me within the next hour or so, I will start preparing a new beta version tonight, after merging #13303 and #13310.

- [ ] Check/Import incoming Translations from Crowdin
- [ ] check/insert correct version name in /main/src/cgeo/geocaching/utils/BranchDetectionHelper.java (using format YYYY.MM.DD-RC)
- [ ] Build release on CI server
- [ ] Perform smoke test with resulting cgeo-RC.apk before continuing
- [ ] Upload and publish cgeo-RC.apk to Google Play
- [ ] Trigger notification to status.cgeo.org via CI server once the release is available on Google Play (not just being processed, but being available for download - sometimes takes up to one day)

**Comment by moving-bits** [comment]
Starting a new beta to include the fixes in #13318, #13319 and #13325.

- [ ] Check/Import incoming Translations from Crowdin
- [ ] check/insert correct version name in /main/src/cgeo/geocaching/utils/BranchDetectionHelper.java (using format YYYY.MM.DD-RC)
- [ ] Build release on CI server
- [ ] Perform smoke test with resulting cgeo-RC.apk before continuing
- [ ] Upload and publish cgeo-RC.apk to Google Play
- [ ] Trigger notification to status.cgeo.org via CI server once the release is available on Google Play (not just being processed, but being available for download - sometimes takes up to one day)

**Comment by moving-bits** [comment]
For what I can see on support, beta version seems to run smooth, so I suggest to create a new release from it. Will do so tomorrow evening, if no one stops me...

- [ ] Check and merge open Crowdin translation pull requests for release
- [ ] Check/Insert correct version name in /main/src/cgeo/geocaching/utils/BranchDetectionHelper.java (Format YYYY.MM.DD) on branch release.
- [ ] Build release on CI server
- [ ] Perform smoke test with resulting cgeo-release.apk before continuing
- [ ] Upload and publish cgeo-release.apk to Google Play
- [ ] Tag the release on Github in format market_yyyymmdd
- [ ] Publish the release on Github including upload of cgeo-release.apk to Github
- [ ] Publish release on F-Droid
- [ ] Trigger notification to status.cgeo.org via CI server once the release is available on Google Play (not just being processed, but being available for download - sometimes takes up to one day)
- [ ] Post release info to Facebook
- [ ] Post release info to Twitter

**Comment by Lineflyer** [comment]
Just today I reported two "beta" issues here:
https://github.com/cgeo/cgeo/issues/13343
https://github.com/cgeo/cgeo/issues/13344

Not sure though if they have been introduced with recent beta or existed before.

**Comment by moving-bits** [comment]
Not sure either, but as Play Store has been fast today and the new release is already out, we should probably put those two issues on a shortlist for next bugfix update. I'll create a new issue for the bugfix release.

**Comment by moving-bits** [comment]
Release has been published a couple of days ago, so let's close here. Thanks to all contributors!
